core/domain/interaction_registry_test.py

```diff
@@ -151,6 +151,8 @@ def traverse_schema_to_find_names(schema):
                     not schema_utils.is_subtitled_unicode_schema(schema) and
                     not schema_utils.is_subtitled_html_schema(schema)
             ):
+                # Because a SubtitledHtml and SubtitledUnicode schema is encoded
+                # as a 'type: dict' schema, we exclude them here.
```

Comment:
Maybe add that they don't have name subfields?

But I'm still not sure I understand. Why not just traverse through them anyway -- is the exclusion really needed?

Reply:
In this particular case, we need to exclude them because they contain camel case names content_id and unicode_str, I've updated the docstr here to reflect that

Reply:
Changed back to custom schema - Done.